<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageDeviceProfilesDialog</class>
 <widget class="QDialog" name="ManageDeviceProfilesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage FocusFinder Profiles</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,10,5">
       <item row="0" column="1">
        <widget class="QComboBox" name="cbxSelectedProfile">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Focus Finder Profile:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnManageProfiles">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Manage Profiles</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layDevices"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
